package.json

```diff
@@ -1,13 +1,18 @@
 {
   "name": "specshell",
-  "version": "0.0.1",
+  "version": "0.0.1-rc.0",
   "description": "Control a program (typically a shell) over standard I/O for testing it",
-  "main": "index.js",
+  "main": "dist/shell.js",
   "scripts": {
     "build": "tsc",
```

Comment:
Possibly a `rm -rf dist/ && tsc` with a separate target of `"build:watch": "tsc -w"`, alternatively as a prepublish target like in https://github.com/binaris/concord/blob/e3238f4590a15d838be75341b292b39c863aa2d0/package.json#L10.

Reply:
Did the latter (and added test-and-lint there, while I'm at it).